src/test/java/com/google/devtools/build/lib/remote/NativeSslTest.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.devtools.build.lib.remote;
+
+import io.netty.handler.ssl.SslProvider;
+import io.netty.handler.ssl.SslContextBuilder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class NativeSslTest {
+  @Test
+  public void nativeSslPresent() throws Exception {
+    SslContextBuilder.forClient().sslProvider(SslProvider.OPENSSL).build();
+  }
```

Comment:
I hope you don't mind me asking but does this actually fail if you mess up the dynamic libraries stripping?

Reply:
Let me answer in two different ways:

* **Immature:** What do you think I am, an AMATEUR? 
* **Mature:** Actually, I *almost* sent this change out without testing it, but yes, it does --  I verified it by removing `.so` files on Linux from the jar, too. Thanks for keeping me honest :)